Description: Fix FTBFS on hurd-i386
 hurd does not define PATH_MAX. Fix taken from pathmax.h
 http://lists.gnu.org/archive/html/bug-gnulib/2011-08/msg00025.html
Author: Daniel Lintott <daniel@serverb.co.uk>
Index: vpcs/src/command.h
===================================================================
--- vpcs.orig/src/command.h	2013-12-04 14:27:08.127058344 +0000
+++ vpcs/src/command.h	2013-12-10 15:59:52.957747281 +0000
@@ -45,6 +45,10 @@
 
 const char *ip4Info(const int id);
 
+#ifndef PATH_MAX
+# define PATH_MAX 8192
+#endif
+
 #endif
 
 /* end of file */
Index: vpcs/src/hv.h
===================================================================
--- vpcs.orig/src/hv.h	2013-09-26 14:50:13.290976827 +0100
+++ vpcs/src/hv.h	2013-12-10 16:01:53.176235930 +0000
@@ -31,6 +31,10 @@
 #include <stdlib.h>
 #include <unistd.h>
 
+#ifndef PATH_MAX
+# define PATH_MAX 8192
+#endif
+
 #define delay_ms(x) usleep((x) * 1000)
 
 #define ERR(out, ...) do { \
